### DISCUSSION TITLE
July 8-14 goals

### DISCUSSION THREAD

**Discussion by eeholmes** [UW-Upwelling-Project/Summer-2022#11] [Team Posts]
Anthony: Play around with the upwelling data, denoise data, explore the wavelet plots and see if it makes sense.

William: Try to identify 1-3 confounders to challenge Bakun's hypothesis. Get sea surface temperature and land temperature off Peru. Make a causal diagrams.

**Comment by zhengx25-1976320** [comment]
Dear Eli, I created a private repo, I just want to double check you can see it?

**Comment by XuweiyiChen** [comment]
Public Repo has been created. Some code has been updated. The upwelling angle has been revised.

Temperature & Upwelling:
https://www.aoml.noaa.gov/phod/docs/wang_weisberg_jgr.pdf

Potential Candidate for Temperature dataset:
https://climateknowledgeportal.worldbank.org/country/peru/climate-data-historical

Is there any better dataset for the sea and land temperature of Peru? @eeholmes 

Very Interesting dataset @zhengx25-1976320 :
https://archive.ics.uci.edu/ml/datasets/El+Nino

**Comment by eeholmes** [comment]
## Anthony

Upwelling signal off coast of Morocco, ts 30 deg latitude
![image](https://user-images.githubusercontent.com/2545978/179322793-aa6d22c0-2b3b-40de-a3d7-0a12dc5b7b0f.png)

at 24 degree latitude
![image](https://user-images.githubusercontent.com/2545978/179322981-c5613002-4bda-4f93-b8be-2780db015fc7.png)

wavelet plot at 30 deg latitude
![image](https://user-images.githubusercontent.com/2545978/179323485-c7366beb-4df5-4fe0-94ac-8d2d211a7b67.png)

wavelet plot at 24 deg latitude
![image](https://user-images.githubusercontent.com/2545978/179323527-83fff819-a5d6-4be9-a735-f68c9a167b2e.png)
